test/system/012-manifest.bats

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bats
+
+load helpers
+
+# Regression test for #8931
+@test "podman images - bare manifest list" {
+    # Create an empty manifest list and list images.
+
+    run_podman inspect --format '{{.ID}}' $IMAGE
+    iid=$output
+
+    run_podman manifest create test:1.0
+    run_podman manifest inspect --insecure $output
```

Comment:
`$output` looks like it'll be the local manifest's ID, which I wouldn't expect us to ask a registry about.  Is this testing the right thing?

Reply:
The `podman login` tests fire up a registry container with TLS configured.  If we did that here, so long as the client isn't configured to trust the CA certificate (HTTPS, but not verifiable), or there is no certificate (plain old HTTP), a client will have to have a working `--insecure` flag in order to be able to talk to it at all.  But we'd have to be looking for an image using a name rather than an ID.